lib/game.rb

```diff
@@ -0,0 +1,7 @@
+class Game
+  def initialize(game_data)
```

Comment:
I believe this line depending on the needs of the methods will be kept the same or changed as well for the other classes we built. Let's regroup near the mid point of our project to determine if we need to add arguments that pass in the CSV data into our initialize methods.